wandb/cli/cli.py

```diff
@@ -2109,30 +2109,14 @@ def put(path, name, description, type, alias, run_id, resume):
         id=run_id,
         resume=resume,
     )
-    # We create the artifact manually to get the current version
```

Comment:
This is really weird. It seems like this was trying to accomplish something beyond just getting the artifact path, but if it was forcing the new artifact to take `latest` I don't think it would work. Any comments on why you think this was here in the first place before you delete it? (but no objections from me)

Reply:
The comment says this was here to get the artifact version, and it is what the code does. I don't know why we didn't just do what I did in the new code. 🤷